lib/daru/vector.rb

```diff
@@ -1437,6 +1475,11 @@ def nan_positions
         end
     end
 
+    # Helper method for rolling_fillna
```

Comment:
I believe the comment, if comment it, the comment could be more helpful, like "Helper method returning validity of arbitrary value" or something like that. Other methods probably could find it useful too ;)